nss-day-cohort-48/fun-website-1-sin-city
# added Dealers names/pics/bios

I added the dealers' names, pics, and bios to dealers.html.

##STEPS TO TEST

1. Pull this branch.
2. Open dealers.html in chrome.
3. Dealers page should display 4 dealers underneath the main dealers pic and intro, along with their names, pictures, and bios.
